test/action_controller/serialization_scope_name_test.rb

```diff
@@ -29,12 +29,12 @@ def render_new_user
 
   def test_default_scope_name
     get :render_new_user
-    assert_equal '{"users":{"admin?":false}}', @response.body
+    assert_equal '{"data":{"admin?":false,"id":"1","type":"test_users"}}', @response.body
```

Comment:
I think that the resource type should stay pluralized as the urls in a rails app are normally pluralized and the jsonapi api spec recommends that the URL for a collection of resources be formed from the resource type.